src/server/verification/verificationAPI.js

```diff
@@ -472,13 +473,15 @@ const setup = (app: Router, verifier: VerificationAPI, gunPublic: StorageAPI, st
 
         if (!user.isEmailConfirmed || isEmailChanged) {
           try {
-            await Mautic.sendVerificationEmail(
-              {
-                ...userRec,
-                mauticId: tempMauticId || mauticId
-              },
-              code
-            )
+            const { fullName } = userRec
+            if (!code || !fullName || !email) {
+              throw new Error('missing input for sending verification email')
+            }
+            const templateData = {
+              firstname: fullName,
```

Comment:
use just first name, see in Mautic.sendVerificationEmail

Reply:
i actually checked, Mautic.sendVerificationEmail uses full name and sends it as first name